packages/libs/components/context/tx.ts

```diff
@@ -146,7 +146,7 @@ export class ReceiptContext {
 
   private constructor() {}
 
-  public get(): EthReceiptKey {
+  get content(): EthReceiptKey {
```

Comment:
Field naming not clear. Align the names, why use `Tx` and `ReceiptContext` within the same scope? What's the difference between these two?

Reply:
sys.context.receipt(ctx).content.unwrap() is batter than sys.context.receipt(ctx).get().unwrap()

Reply:
better to be sys.context.receipt(ctx).unwrap()